mbtiles.go

```diff
@@ -11,17 +11,32 @@ import (
 
 type MbTilesMetadata map[string]string
 
+type MbTilesFormat string
+
+const (
+	Pbf  MbTilesFormat = "pbf"
```

Comment:
By go conventions, these should be prefixed with the type. So this would become `MbTilesFormatPbf` and the row below as well.

Reply:
done